.github/workflows/examples.yml

```diff
@@ -38,6 +38,8 @@ env:
   MLFLOW_HOME: ${{ github.workspace }}
   MLFLOW_CONDA_HOME: /usr/share/miniconda
   PYTHONFAULTHANDLER: "1"
+  MLFLOW_ENV_MANAGER: uv
+  UV_CACHE_DIR: /tmp/uv-cache
```

Comment:
what is this change for?

Reply:
to run the examples with UV instead of the default that is currenlty used for examples - virtualenv